lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
They are global and per-call settings.  The global ones apply on every query unless overridden in the call.